setup.py

```diff
@@ -11,12 +11,12 @@
     license='MIT',
     description='Very flexible date parsing and normalization utilities',
     long_description=__long_description__,
-    author='Open Knowledge Foundation',
-    author_email='info@okfn.org',
-    url='http://github.com/okfn/flexidate/',
```

Comment:
Why is the url changing here? This will still live at github.com/okfn/flexidate ...